pep-0701.rst

```diff
@@ -251,7 +251,7 @@ token contents. However, incorporating this feature into a given parser
 implementation requires the lexer to keep track of the raw string contents of
 the expression part of the f-string and make them available to the parser when
 the parse tree is constructed for f-string nodes. A pure "untokenization" is not
-enough because as specified currently, f-string debugging preserve whitespace,
+enough because as specified currently, f-string debugging preserves whitespace,
```

Comment:
I think this would be a little clearer:
```suggestion
enough because as specified currently, f-string debug expressions preserve whitespace in the expression,
```

Reply:
It could also be nice to explicitly mention whether comments are preserved. (I wasn't quite sure of whether they were or not; the reference implementation currently doesn't allow any comments)